tests/avifyuv.c

```diff
@@ -164,13 +164,19 @@ int main(int argc, char * argv[])
                         avifRGBImageSetDefaults(&srcRGB, image);
                         srcRGB.format = AVIF_RGB_FORMAT_RGB;
                         srcRGB.depth = rgbDepth;
-                        avifRGBImageAllocatePixels(&srcRGB);
 
                         avifRGBImage dstRGB;
                         avifRGBImageSetDefaults(&dstRGB, image);
                         dstRGB.format = AVIF_RGB_FORMAT_RGB;
                         dstRGB.depth = rgbDepth;
-                        avifRGBImageAllocatePixels(&dstRGB);
+
+                        if ((avifRGBImageAllocatePixels(&srcRGB) != AVIF_RESULT_OK) ||
+                            (avifRGBImageAllocatePixels(&dstRGB) != AVIF_RESULT_OK)) {
+                            avifRGBImageFreePixels(&srcRGB);
+                            avifImageDestroy(image);
```

Comment:
Optional: It takes a little thought to understand why `avifRGBImageFreePixels(&dstRGB)` is not needed. May want to rewrite this in the same way as the code below, at the cost of an extra if statement.